docs/route-tasks.md

```diff
@@ -0,0 +1,171 @@
+---
+id: route-tasks
+title: How to route Tasks
+sidebar_label: Route Tasks
+---
+
+Consider the specifications of a Worker:
+- A Worker only handles [Activity](docs/learn-activities) and [Workflow](docs/learn-workflows) Tasks that it is registered to handle. This means that a Task must be paired with a Worker that is registered to handle that Task.
+- A Worker runs on a specific host. This means that Activity Tasks that depend on host-specific resources, such as a local file, must be executed by a Worker on that host.
+
+The question is how do we make sure a specific Worker is executing a specific Task? We do this through "Task routing".
+
+## Task routing with Task Queues
+
+Task Queues are what enable "Task routing". Task routing is the development technique that pairs Tasks with Workers. The basics of it can be broken down into 3 steps:
+
+1. Create a Worker that is registered to handle Workflow and/or Activity Tasks.
+2. Subscribe the Worker to listen to a Task Queue.
+3. Configure Workflows and Activities to send Tasks to the Task Queue.
+
+Let's look at the ["Hello World" sample in Go](https://github.com/temporalio/temporal-go-samples/tree/master/helloworld) to see how this is done.
+
+In the following code snippet, we [create a new Worker](https://github.com/temporalio/go-samples/blob/master/helloworld/worker/main.go#L26) that is subscribed to the "hello-world" Task Queue and is registered to handle the `helloworld.Workflow` and `helloworld.Activity`:
+
+```go
+// Worker is created and subscribed to the "hello-world" Task Queue
+w := worker.New(c, "hello-world", worker.Options{})
+defer w.Stop()
+// Worker is registered to handle the helloworld Workflow
+w.RegisterWorkflow(helloworld.Workflow)
+// Worker is registered to handle the helloworld Activity
+w.RegisterActivity(helloworld.Activity)
+```
+
+In the next code snippet, we [start the `helloworld.Workflow`](https://github.com/temporalio/go-samples/blob/master/helloworld/starter/main.go#L25):
```

Comment:
Again where would this reside

Reply:
Where it resides definitely depends on the SDK language and app design - I would like to experiment with using a "Sample source reference" link that would send the user to the exact code lines in the sample that best reflects the snippet (the actual go samples need to be updated to be better samples for the docs - part of the sync project)